### PR TITLE
[MU4] volume_level_slider

### PR DESCRIPTION
**Pull request by vpereverzev**
- Implemented a new VolumeSlider component
- Fixed an appearance of KnobControl and VolumePressureMeter
- Turned on dynamic sample loading in Fluid synth

Peek 2021-07-14 20-44

### CODE REVIEW

File: src/framework/audio/internal/synthesizers/fluidsynth/fluidsynth.cpp [hunk 1]
```diff
@@ -112,7 +112,8 @@ Ret FluidSynth::init()
     fluid_settings_setint(m_fluid->settings, "synth.audio-channels", FLUID_AUDIO_CHANNELS_PAIR); // 1 pair of audio channels
     fluid_settings_setint(m_fluid->settings, "synth.lock-memory", 0);
     fluid_settings_setint(m_fluid->settings, "synth.threadsafe-api", 0);
-    fluid_settings_setint(m_fluid->settings, "synth.midi-channels", 80);
+    fluid_settings_setint(m_fluid->settings, "synth.midi-channels", 16);
```
Comment (igorkorsukov) [inline src/framework/audio/internal/synthesizers/fluidsynth/fluidsynth.cpp:115]: 16 is very little for us :)

File: src/framework/audio/qml/MuseScore/Audio/VolumePressureMeter.qml [hunk 2]
```diff
@@ -33,14 +33,11 @@ Canvas {
...
-    width: 24
+    width: 6
     height: 140
 
-    renderStrategy: Canvas.Cooperative
-    renderTarget: Canvas.FramebufferObject
-
     QtObject {
-        id: privateProperties
+        id: internals
```
Comment (igorkorsukov) [inline src/framework/audio/qml/MuseScore/Audio/VolumePressureMeter.qml:40]: usually called `prv` in other places

File: src/framework/audio/qml/MuseScore/Audio/VolumeSlider.qml [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+    width: 32 + internals.unitsTextWidth
+
+    from: -48
+    to: 12
+    value: 0
+    stepSize: 0.1
+    orientation: Qt.Vertical
+
+    QtObject {
+        id: internals
```
Comment (igorkorsukov) [inline src/framework/audio/qml/MuseScore/Audio/VolumeSlider.qml:41]: usually called prv in other places

File: src/framework/audio/internal/synthesizers/fluidsynth/fluidsynth.cpp (hunk 1 above)
Comment (vpereverzev) [reply src/framework/audio/internal/synthesizers/fluidsynth/fluidsynth.cpp:115]: it's enough for a single track, assuming that we have synth instance per track

Comment (igorkorsukov) [reply src/framework/audio/internal/synthesizers/fluidsynth/fluidsynth.cpp:115]: oh, sure

File: src/framework/audio/qml/MuseScore/Audio/VolumePressureMeter.qml (hunk 2 above)
Comment (vpereverzev) [reply src/framework/audio/qml/MuseScore/Audio/VolumePressureMeter.qml:40]: ok, why not

I'll rename it in the next PR

**Pull request merged**